src/actions/JobsAction.ts

```diff
@@ -4,25 +4,36 @@ import {
   JOBS_FAIL,
   JOBS_LOADING,
   JOBS_SUCCESS,
+  SEARCH_FILTER,
   JobDispatchTypes,
 } from '../actions/JobsActionType';
 
-export const GetJob = () => async (dispatch: Dispatch<JobDispatchTypes>): Promise<void> => {
-  try {
-    dispatch({
-      type: JOBS_LOADING,
-    });
-    const res = await axios.get(
-      `https://6239311d043817a543df0a3f.mockapi.io/api/jobs`
-    );
+export const BASE_URL = 'https://6239311d043817a543df0a3f.mockapi.io/api/jobs';
 
-    dispatch({
-      type: JOBS_SUCCESS,
-      payload: res.data,
-    });
-  } catch (e) {
-    dispatch({
-      type: JOBS_FAIL,
-    });
-  }
-};
+export const getJob =
+  (searchTitle: string) =>
+  async (dispatch: Dispatch<JobDispatchTypes>): Promise<void> => {
+    try {
+      dispatch({
+        type: JOBS_LOADING,
+      });
+      const res = await axios.get(BASE_URL + `?name=${searchTitle}`);
```

Comment:
- Please use `template string` for this. 
- When `searchTitle` is null, endpoint should be `BASE_URL` instead `BASE_URL?name=`

Reply:
Is it fixed?